.circleci/config.yml

```diff
@@ -17,12 +17,12 @@ jobs:
             SHASNIP=$(echo $CIRCLE_SHA1 | cut -c1-7)
             cat > dashboard.cmake << EOF
             set(CTEST_SITE "CircleCI")
-            set(CTEST_BUILD_NAME "External-LabelErodeDilate-${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM}-${SHASNIP}")
+            set(CTEST_BUILD_NAME "External-ITKLabelErodeDilate-${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM}-${SHASNIP}")
```

Comment:
Changes in this file are not consistent with the ModuleTemplate:
https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/bb692c7526016ff3729eda2a2f05b29b160f338e/%7B%7Bcookiecutter.project_name%7D%7D/.circleci/config.yml#L20

The name of the repository is `ITKLabelErodeDilate`, as it should be. And http(s) links to it need to reflect that. But the name of the module is `LabelErodeDilate`, as external modules do not have 'ITK` prefix as part of their name.

Reply:
> Changes in this file are not consistent with the ModuleTemplate:
https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/bb692c7526016ff3729eda2a2f05b29b160f338e/%7B%7Bcookiecutter.project_name%7D%7D/.circleci/config.yml#L20

This is using `cookiecutter.project_name` (contains *ITK*) versus `cookiecutter.module_name`, which does not.  The defaults are computed here:

https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/bb692c7526016ff3729eda2a2f05b29b160f338e/cookiecutter.json#L5-L6